emulator/pendingBlock.go

```diff
@@ -126,6 +126,20 @@ func (b *pendingBlock) Collections() []*flowgo.LightCollection {
 	return []*flowgo.LightCollection{&collection}
 }
 
+func (b *pendingBlock) FullCollections() []*flowgo.Collection {
+	if len(b.transactionIDs) == 0 {
+		return []*flowgo.Collection{}
+	}
+
+	transactions := make([]*flowgo.TransactionBody, len(b.transactionIDs))
+	for i, txID := range b.transactionIDs {
+		transactions[i] = b.transactions[txID]
+	}
```

Comment:
Maybe just use `copy`?

Reply:
How to `copy`? It's reading a transaction by id from a map.

Reply:
I think this approach is needed to preserve the order

Reply:
Ah, sorry, I didn't realize the input was a map